### ISSUE TITLE
IDISP007 should not warn here

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/IDisposableAnalyzers#256]
```cs
        [OneTimeSetUp]
        public void OneTimeSetUp()
        {
            _container = Factory.Create();
        }

        [OneTimeTearDown]
        public void OneTimeTearDown()
        {
            _container.Dispose();
        }
```

Factory is in another project, add assumption test.
